src/lib/modernExtend.ts

```diff
@@ -1836,7 +1837,7 @@ export function electricityMeter(args?: ElectricityMeterArgs): ModernExtend {
                             // Check if this property has a divisor and multiplier, e.g. AC frequency doesn't.
                             if ('divisor' in property) {
                                 // In case multiplier or divisor was provided, use that instead of reading from device.
-                                if (property.forced) {
+                                if (property.forced && property.forced.divisor) {
```

Comment:
```suggestion
                                if (property.forced?.divisor || property.forced?.multiplier) {
```

Reply:
Even with the optional property chaining the compiler complains that: 

```
Property 'divisor' does not exist on type 'false | MultiplierDivisor | (MultiplierDivisor & { min?: ReportingConfigTime; })'.
  Property 'divisor' does not exist on type 'false'.ts(2339)
```

Only fix I could find was to include a test for forced:
```
if (property.forced && (property.forced.divisor || property.forced.multiplier)) {
```